src/dropdown.styles.ts

```diff
@@ -2,11 +2,13 @@ import { css } from 'lit';
 import focusOutline from './styles/focus-outline.js';
 import opacityAndScaleAnimation from './styles/opacity-and-scale-animation.js';
 import visuallyHidden from './styles/visually-hidden.js';
+import skeleton from './styles/skeleton.js';
 
 export default [
   css`
     ${focusOutline('.add-button:focus-visible')}
     ${opacityAndScaleAnimation('.options-and-footer:popover-open')}
+    ${skeleton('.loading', '.bar')}
```

Comment:
These make-shift mixins are actually pretty flexible. If we later a another type of skeleton, we can either rename this one to "loading" or else change its API:

```ts
${skeleton({ 
  selectors: { 
    container: string, 
    bar: string 
  }, 
  type?:  'simple' | 'advanced' = 'simple'
})}
```